docs/topics/db/examples/one_to_one.txt

```diff
@@ -149,3 +149,29 @@ Query the waiters::
     <QuerySet [<Waiter: Joe the waiter at Demon Dogs the restaurant>]>
     >>> Waiter.objects.filter(restaurant__place__name__startswith="Demon")
     <QuerySet [<Waiter: Joe the waiter at Demon Dogs the restaurant>]>
+
+As the relation is defined on the Restaurant side, if a place is deleted it will delete its restaurant:
```

Comment:
I would move it before `Add a Waiter to the Restaurant::` and add a reference to the `on_delete`. For example:
```diff
diff --git a/docs/topics/db/examples/one_to_one.txt b/docs/topics/db/examples/one_to_one.txt
index 3edf866c6e..92f5e9ce40 100644
--- a/docs/topics/db/examples/one_to_one.txt
+++ b/docs/topics/db/examples/one_to_one.txt
@@ -137,6 +137,16 @@ This also works in reverse::
     >>> Place.objects.get(restaurant__place__name__startswith="Demon")
     <Place: Demon Dogs the place>
 
+If you delete a place, its restaurant will be deleted (assuming that the
+``OneToOneField`` was defined with
+:attr:`~django.db.models.ForeignKey.on_delete` set to ``CASCADE``, which is the
+default)::
+
+    >>> p2.delete()
+    (2, {'one_to_one.Restaurant': 1, 'one_to_one.Place': 1})
+    >>> Restaurant.objects.all()
+    <QuerySet [<Restaurant: Demon Dogs the restaurant>]>
+
 Add a Waiter to the Restaurant::
 
     >>> w = r.waiter_set.create(name='Joe')
```